dist/index.js

```diff
@@ -90,14 +90,26 @@ function dottedQuadFileVersion(version) {
   return version;
 }
 
+function get_fix_versions(v) {
+  if (
+    v.vulnerability.fix &&
```

Comment:
🐛 probably need to check `.fix.state === "fixed"` based on: https://github.com/anchore/grype-db/blob/main/pkg/db/v3/fix.go#L5-L10